test-case/check-alsabat.sh

```diff
@@ -64,11 +72,52 @@ function __upload_wav_file
     done
 }
 
+function set_pga_to_unity_gain
```

Comment:
`function` is less portable across shells and adds no value.
For now we only support bash but it'd be great to be compatible with more.

The most portable is:
```shell
set_pga_to_unity_gain()
{
  ...
```

Reply:
Ah, yes I was following the convention in the script, agree. Maybe another PR could be done to update the rest.